packages/amplify-provider-awscloudformation/src/aws-utils/aws-pinpoint.js

```diff
@@ -66,6 +66,7 @@ function getPinpointRegionMapping() {
     'ap-northeast-1': latestPinpointRegions >= 1 ? 'ap-northeast-1' : 'us-west-2',
     'eu-central-1': 'eu-central-1',
     'eu-north-1': 'eu-central-1',
+    'eu-south-1': 'eu-central-1',
```

Comment:
Pinpoint is not available in eu-south-1. Falling back to eu-central-1.

https://docs.aws.amazon.com/general/latest/gr/pinpoint.html